src/libs/actions/BankAccounts.js

```diff
@@ -57,10 +57,10 @@ function clearOnfidoToken() {
 
 /**
  * Helper method to build the Onyx data required during setup of a Verified Business Bank Account
- *
+ * @param {String | undefined} currentStep
```

Comment:
Can you update the doc to explain why it's valid to pass `undefined` and what we should expect to happen in this strange case?